src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+  private static final String COMMAND_TODO = "todo";
+  private static final String COMMAND_DEADLINE = "deadline";
+  private static final String COMMAND_EVENT = "event";
+  private static final String COMMAND_LIST = "list";
+  private static final String COMMAND_DONE = "done";
+  private static final String COMMAND_BYE = "bye";
+
+  public static void main(String[] args) {
+    String logo =
+        " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|";
+    String line = "\n - - - - - - - - - - - - - - - - - - - - - - - -  \n";
+    String line2 = System.lineSeparator() + "|    ";
+    String description;
+    ArrayList<Task> taskList = new ArrayList<Task>();
+    System.out.println("\nHello from\n" + logo);
+
+    System.out.println(
+        String.format(
+            line + "|             %-34s|\n|       %-40s|",
+            "Hello! I'm Duke :D", "What can I do for you today?") + line);
+
+   // System.out.print(line2 + line + "Hello! I'm Duke :D" + line + "What can I do for you today?" + "\n" + line2);
+    Scanner scan = new Scanner(System.in);
+
+    while (true) {
+
+      String input = scan.nextLine();
+      final Command command = splitCommanandArgs(input);
+
+      switch (command.getCommandName()) {
+        case COMMAND_BYE: // exit program
```

Comment:
shouldnt case statement be aligned with the switch statement according to our coding standards?